src/main/java/losty/netatmo/NetatmoParseUtils.java

```diff
@@ -252,17 +252,27 @@ public static Home parseHomestatus(final JSONObject response) throws JSONExcepti
             final float therm_measured_temperature = jsonRoom.getFloat("therm_measured_temperature");
             final float therm_setpoint_temperature = jsonRoom.getFloat("therm_setpoint_temperature");
             final String therm_setpoint_mode = jsonRoom.getString("therm_setpoint_mode");
-            final long therm_setpoint_start_time = jsonRoom.getLong("therm_setpoint_start_time");
-            final long therm_setpoint_end_time = jsonRoom.getLong("therm_setpoint_end_time");
+
+            Long therm_setpoint_start_time = null;
+            if (jsonRoom.opt("therm_setpoint_start_time") != null) {
+                therm_setpoint_start_time = jsonRoom.getLong("therm_setpoint_start_time");
+            }
+
+            Long therm_setpoint_end_time = null;
+            if (jsonRoom.opt("therm_setpoint_end_time") != null) {
+                therm_setpoint_end_time = jsonRoom.getLong("therm_setpoint_end_time");
+            }
 
             Room room = new Room();
             room.setId(room_id);
             room.setReachable(reachable);
             room.setTherm_measured_temperature(therm_measured_temperature);
             room.setTherm_setpoint_temperature(therm_setpoint_temperature);
             room.setTherm_setpoint_mode(therm_setpoint_mode);
-            room.setTherm_setpoint_start_time(therm_setpoint_start_time);
-            room.setTherm_setpoint_end_time(therm_setpoint_end_time);
+            if (therm_setpoint_start_time != null){
+                room.setTherm_setpoint_start_time(therm_setpoint_start_time);}
```

Comment:
This is a bit odd - not setting the value woild just mean the default value "0L" remains.

Hmm... we could use...
```
final long therm_setpoint_start_time = jsonRoom.optLong("therm_setpoint_start_time", Long.MIN_VALUE)`;
```

...but I rather think switching `Room`class to have proper `Long` objects makes more sense here, does it?

...and BTW: are there any other paramters that might be null?

Reply:
Hi! I like your second option. I didn't change it before because I didn't want to change your model. 
I don't know what other params might be null but the solution could be the same.

I'll pull another request with the new fix

Reply:
Yes, in-deed... as this will also change the interface...

...but with autoboxing long -> Long should not harm too much and it's the only way to clearly mark it could be null, I'd say...